CONTRIBUTING.md

```diff
@@ -54,7 +54,7 @@ To run the SQL tests locally you will need to:
 
 1. Setup a database using the server(s) that you want to test against.
 1. Create the database schema using the `DatabaseSchema-<engine>.sql` file.
-1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](docs/configuration-file.md#accessing-environment-variables).
+1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/en-us/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

Comment:
```suggestion
1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

removing `en-us` will allow browser to determine locale for non-US based readers.